superset-embedded-sdk/src/index.ts

```diff
@@ -64,6 +64,8 @@ export type EmbedDashboardParams = {
   iframeTitle?: string
   /** additional iframe sandbox attributes ex (allow-top-navigation, allow-popups-to-escape-sandbox) **/
   iframeSandboxExtras?: string[]
+  /** force a specific refererPolicy to be used in the iframe request **/
+  referrerPolicy?: ReferrerPolicy | null
```

Comment:
nit: if already optional and null is only used as "is set?" you should be fine w/o it like all others optionals here. You wouldn't need the default = null neither.

Reply:
great catch! thanks for the suggestion. I've just implemented this change in a new commit